.github/workflows/create-release.yaml

```diff
@@ -102,3 +108,19 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: ./scripts/publish_release.sh ${{ needs.create-draft.outputs.release_id }}
+
+  notify:
+    runs-on: ubuntu-latest
+    needs: publish-release
+    if: success()
```

Comment:
because in the card only notification when release is *done* is mentioned. Not whether it failed or succeeded. And if any of the tests failed, release is not done.

Reply:
Ok 🤷‍♂️